src/mame/video/decbac06.cpp

```diff
@@ -343,7 +343,7 @@ void deco_bac06_device::custom_tilemap_draw(bitmap_ind16 &bitmap,
 
 			src_x++;
 			// TODO : Convertable masking feature into set_transmask related?
-			if ((flags & TILEMAP_DRAW_OPAQUE) || (p & bppmask))
```

Comment:
1. The official way to use the TODO syntax is `// TODO:`, without the space between TODO and : cfr. [Microsoft](https://docs.microsoft.com/en-us/visualstudio/ide/using-the-task-list?view=vs-2019) article
2. I don't know implemention detail here, but I suppose the right comment is `Convert masking feature with set_transmask fn?`
3. is_drawed -> is_drawn